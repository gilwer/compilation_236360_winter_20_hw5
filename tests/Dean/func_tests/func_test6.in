
int foo(int x,int y, bool magzim){
	print("dont be magzim");

	int a=10;
	int c=100;
	if(magzim)
	{
		return (x*y)*c;//FIX
		}
	else{
		return (x+y)*a; // FIX
		}
}


void main(){
	int x = foo(5,3,false);
	printi(x);
	int y = foo(50,30,true);
	printi(y);
	
}