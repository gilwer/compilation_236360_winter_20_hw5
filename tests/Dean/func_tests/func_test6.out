dont be magzim
80
dont be magzim
150000
